### DISCUSSION TITLE
چند مشکل و پیشنهاد

### DISCUSSION THREAD

**Discussion by hreza137575** [amir1376/ab-download-manager#233] [Ideas]
تنظیماتی که باید اضافه شود تا حرفی برای گفتن در مقابل idm داشته باشد.

پیشنهادات و مشکلات:
اضافه کردن سبد پرتاب ، لینک کشیده شود و در سبد رها شود
وقتی محدود کردن سرعت دانلود رو فعال می کنیم و مثلا روی 600 کیلوبایت می ذاریم؛ سری بعد دوباره باید خودش پیشفرض روی 600 ست شده باشه، نه اینکه دوباره بشه 10 کیلوبایت

تنظیمات زیر به صف دانلود اضافه شود:
بستن اجباری برنامه های ویندوز و خاموش کردن سیستم بعد از اتمام دانلود
محدود کردن سرعت دانلود

امکانات پیشنهادی:
اولویت دادن سرعت به برنامه های دیگر؛ یعنی زمانی که فایلی در حال دانلود هستش، ولی کاربر در حال وب گردی هستش، اولویت رو بده به وب گردی و سرعت دانلود رو اتومات پایین بیاره؛ البته انتخاب این گزینه اختیاری باشه

**Comment by Mahantor** [comment]
نرم افزار خوبیه ولی تا زمانیکه قابلیت های idm روش پیاده سازی نشه به درد کاربر نمی خوره ، توسعه دهنده هم برای اضافه نکردن این قابلیت ها خیلی مقاومت میکنه ، توی انجمن های مختلف که نظرات میبینی خیلیا میان یه مدت استفاده میکنن و میزارنش کنار چون آپشن ها و دسترسی هایی که تو idm داشتن توی ab download manager نمی بینن

**Reply by amir1376** [reply]
> نرم افزار خوبیه ولی تا زمانیکه قابلیت های idm روش پیاده سازی نشه به درد کاربر نمی خوره ، توسعه دهنده هم برای اضافه نکردن این قابلیت ها خیلی مقاومت میکنه ، توی انجمن های مختلف که نظرات میبینی خیلیا میان یه مدت استفاده میکنن و میزارنش کنار چون آپشن ها و دسترسی هایی که تو idm داشتن توی ab download manager نمی بینن

برنامه نسبتا جدید هست و یک چیز کاملا طبیعی هست که ایراداتی داشته باشه. که خب به مرور چیز های جدید اضافه میشن

در خصوص این که میگین "خیلی مقاومت میکنه به اضافه نکردن قابلیت های جدید" این یک جمله بصورت کلی هست! 
یک سری ویژگی ها امکان اضافه شدنش به این پروژه وجود نداره با انوع دلیل ها از نقض کپی رایت گرفته تا هزینه محور بودن و ...

اگر مورد خاصی مد نظر هست بفرمایین تا دلیلش رو بگم.

**Comment by Daniyal09** [comment]
![{32C25AE1-77CF-482B-9BB2-C89BD2C59F94}](https://github.com/user-attachments/assets/087e286a-8672-4246-b900-e59da10ab925)
لطفا این را هم اضافه کنید

**Reply by amir1376** [reply]
> ![{32C25AE1-77CF-482B-9BB2-C89BD2C59F94}](https://github.com/user-attachments/assets/087e286a-8672-4246-b900-e59da10ab925)
> لطفا این را هم اضافه کنید 

این برنامه فایل هارو مستقیما روی مقصد نهایی ذخیره میکنه و کلا مسیر  temp  نداره

**Comment by amir1376** [comment]
> تنظیماتی که باید اضافه شود تا حرفی برای گفتن در مقابل idm داشته باشد.
> 
> پیشنهادات و مشکلات: اضافه کردن سبد پرتاب ، لینک کشیده شود و در سبد رها شود وقتی محدود کردن سرعت دانلود رو فعال می کنیم و مثلا روی 600 کیلوبایت می ذاریم؛ سری بعد دوباره باید خودش پیشفرض روی 600 ست شده باشه، نه اینکه دوباره بشه 10 کیلوبایت
> 
> تنظیمات زیر به صف دانلود اضافه شود: بستن اجباری برنامه های ویندوز و خاموش کردن سیستم بعد از اتمام دانلود محدود کردن سرعت دانلود
> 
> امکانات پیشنهادی: اولویت دادن سرعت به برنامه های دیگر؛ یعنی زمانی که فایلی در حال دانلود هستش، ولی کاربر در حال وب گردی هستش، اولویت رو بده به وب گردی و سرعت دانلود رو اتومات پایین بیاره؛ البته انتخاب این گزینه اختیاری باشه


سلام وقت بخیر ممنونم از پیشنهاداتتون بعضی ها به برنامه اضافه شده اند.  بعضی ها هم در صف انجام هستن.
